src/engine/client/cl_keys.cpp

```diff
@@ -43,7 +43,7 @@ key up events are sent even if in console mode
 
 */
 
-bool key_overstrikeMode;
+static bool overstrikeModeOn;
 bool bindingsModified;
```

Comment:
Can this be made static too?

Reply:
Well, it is used somewhere, in Com_WriteConfiguration.